dev/com.ibm.ws.concurrent.persistent_fat/build.gradle

```diff
@@ -19,7 +19,7 @@ configurations {
 
 dependencies {
   //Test container libraries
-  requiredLibs 'org.testcontainers:testcontainers:1.12.0',
+  requiredLibs 'org.testcontainers:testcontainers:1.12.1',
                'org.testcontainers:database-commons:1.12.0',
```

Comment:
lets upgrade all testcontainers components to 1.12.3 (i.e. everything in the `org. testcontainers:testcontainers` group), not just the core `org.testcontainers:testcontainers` module